### PR TITLE
#443:feat:news-page

### PR DESCRIPTION
**Pull request by yegobaroni**
# Description

This pull request works on issue #443 which reuses issue #411

Fixes # 443

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Screen Shot 2023-03-10 at 10 35 58 PM

mobile

Screen Shot 2023-03-10 at 10 36 19 PM

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/components/ui-lists/src/lib/elewa-group-article-list/elewa-group-article-list.component.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import { Component, Input } from '@angular/core';
+export interface Article { image: string; timestamp: string; title: string; body: string; topic: string;} 
+
```
Comment (IanOdhiambo9) [inline libs/features/components/ui-lists/src/lib/elewa-group-article-list/elewa-group-article-list.component.ts:3]: Move this to an interface file in the same lib models/articles.interface.ts

File: libs/pages/elewa/news/src/lib/components/elewa-news-section/elewa-news-section.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div class="elewa-news-section-container">
+    <h2>News & Stories</h2>
+    <nav class="elewa-news-section-items">
+      <a class="elewa-news-section-item" (click)="selectedTab='all-news'; filterArticles();" [class.active]="selectedTab==='all-news'">All News</a>
+      <a class="elewa-news-section-item" (click)="selectedTab='elewa'; filterArticles();" [class.active]="selectedTab==='elewa'">Elewa</a>
+      <a class="elewa-news-section-item" (click)="selectedTab='italanta'; filterArticles();" [class.active]="selectedTab==='italanta'">Italanta</a>
+      <a class="elewa-news-section-item" (click)="selectedTab='press'; filterArticles();" [class.active]="selectedTab==='press'">Press</a>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/news/src/lib/components/elewa-news-section/elewa-news-section.component.html:7]: use a loop and make this a button

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Your fonts and font sizes are not as provided on the design. Please check

**Comment by IanOdhiambo9** [comment]
The tabs should remain tabs on mobile. Just make them a grid of 2 x 2

**Comment by yegobaroni** [comment]
@IanOdhiambo9 changes made. kindly review.

**Pull request closed without merging**